site/zenodo_rdm/legacy/serializers/schemas.py

```diff
@@ -63,6 +66,96 @@ def dump_resource_type(self, result, original, **kwargs):
                 result[f"{upload_type}_type"] = resource_type_id.split("-")[-1]
         return result
 
+    def _funder(self, funder):
+        """Serialize RDM funder into Zenodo legacy funder."""
+        legacy_funder = {"name": funder["name"]}
+
+        f_identifiers = {}
+        for identifier in funder.get("identifiers"):
+            scheme = identifier["scheme"]
+            if scheme == "oaf":
+                f_identifiers["oaf"] = identifier["identifier"]
+
+            if scheme == "doi":
+                legacy_funder["doi"] = identifier["identifier"]
+
+        if f_identifiers:
+            legacy_funder["identifiers"] = f_identifiers
+
+        value = funder.get("country")
+        if value:
+            legacy_funder["country"] = value
+
+        return legacy_funder
+
+    def _award(self, award):
+        """Serialize an RDM award into a legacy Zenodo grant."""
+        legacy_grant = {
+            "code": award["number"],
+            "internal_id": f"{award['funder']['id']}::{award['id']}",
```

Comment:
This is part of the questions of the issue:
a) do we change format (break backwards compatibility) and do ror::award-id
b) we can implement an inverse dict and translate to funder doi. If this is the case, what do we do with those new ones not in the mapping?

Reply:
`b)` is the way, we reverse map, and in case there's no mapping we maintain the RDM-style i.e. `ror-funder::award-code`